elastic4s-domain/src/main/scala/com/sksamuel/elastic4s/requests/indexlifecyclemanagement/policy/IndexLifecyclePolicy.scala

```diff
@@ -0,0 +1,49 @@
+package com.sksamuel.elastic4s.requests.indexlifecyclemanagement.policy
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.sksamuel.elastic4s.JacksonSupport
+import com.sksamuel.elastic4s.json.{JsonValue, StringValue}
+
+import scala.collection.JavaConverters._
+
+case class IndexLifecyclePolicy(
+    name: String,
+    phases: List[IndexLifecyclePolicyPhase],
```

Comment:
I think maybe a `Map` is more appropriate here? Then you don't need to override the equals method.